### PR TITLE
fix mobile burger menu, fix ipad main menu, add title for price, chan…

### PR DESCRIPTION
**Pull request by LucyHrinishyna**
…ge cursor on disabled button, fix filter, fix locale

Fixes #

### Changes

Summary of changes you made.

* Change 1
* Change 2

### Test plan

Explain how to test changes you made.

* [ ] Step 1
* [ ] Step 2

### Demo (optional)

Sreenshots (before & after) or video that show result of your work.

### CODE REVIEW

File: src/client/Wrapper.js [hunk 1]
```diff
@@ -278,6 +281,7 @@ export default class Wrapper extends React.PureComponent {
         <ConfigProvider locale={enUS}>
           <UsedLocaleContext.Provider value={usedLocale}>
             <Layout data-dir={language && language.rtl ? 'rtl' : 'ltr'}>
+              route
```
Comment (milklim) [inline src/client/Wrapper.js]: What this? Why do we need it?

Comment (LucyHrinishyna) [reply src/client/Wrapper.js]: we don`t need this, i remove it

File: src/investarena/components/QuickForecastPage/QuickForecastCard/QuickForecastCard.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import { injectIntl } from 'react-intl';
+import { injectIntl, FormattedRelative } from 'react-intl';
```
Comment (milklim) [inline src/investarena/components/QuickForecastPage/QuickForecastCard/QuickForecastCard.js]: unused method - remove

File: src/investarena/components/QuickForecastPage/QuickForecastCard/QuickForecastCard.js [hunk 3]
```diff
@@ -129,19 +133,22 @@ const QuickForecastCard = ({
...
-                  src={avatar}
-                  alt={predictionObjectName}
-                />
+                <Link className="ForecastCard__link" to={`/object/${link}`}>
+                  <img
+                    className="ForecastCard__img ForecastCard__img--little"
+                    src={avatar}
+                    alt={predictionObjectName}
+                  />
+                  &#160;
```
Comment (milklim) [inline src/investarena/components/QuickForecastPage/QuickForecastCard/QuickForecastCard.js]: better to use &nbsp;

File: src/investarena/components/QuickForecastPage/QuickForecastPage.js [hunk 4]
```diff
@@ -63,146 +65,188 @@ const QuickForecastPage = props => {
...
+                  })}
+                  :&#160;
+                  <USDDisplay value={props.roundInformation.rewards} />
+                </span>
+                <span className="reward__row">
+                  {props.intl.formatMessage({
+                    id: 'forecast_round',
+                    defaultMessage: 'Current round',
+                  })}
+                  :&#160;
```
Comment (milklim) [inline src/investarena/components/QuickForecastPage/QuickForecastPage.js]: "&nbsp" is more readable

File: src/investarena/components/QuickForecastPage/QuickForecastPage.js [hunk 5]
```diff
@@ -63,146 +65,188 @@ const QuickForecastPage = props => {
...
-          )}
+          <Affix className="rightContainer" stickPosition={122}>
+            <div className="right">
+              <div className="reward">
+                <span className="reward__row">
+                  {props.intl.formatMessage({
+                    id: 'forecasts_rewards',
+                    defaultMessage: 'Rewards',
+                  })}
+                  :&#160;
```
Comment (milklim) [inline src/investarena/components/QuickForecastPage/QuickForecastPage.js]: "&nbsp" is more readable

File: src/investarena/components/QuickForecastPage/QuickForecastPage.js [hunk 6]
```diff
@@ -63,146 +65,188 @@ const QuickForecastPage = props => {
...
-              :&#160;
-              <USDDisplay value={props.roundInformation.voitingPowers} />
-            </span>
+          </Affix>
+          <div className="center">
+            {isLoading ? (
+              <React.Fragment>
+                <div className="timer-container">
+                  <Icon type="clock-circle" />
+                  &#160;
```
Comment (milklim) [inline src/investarena/components/QuickForecastPage/QuickForecastPage.js]: "&nbsp" is more readable

File: src/investarena/components/QuickForecastPage/QuickForecastPage.js [hunk 7]
```diff
@@ -63,146 +65,188 @@ const QuickForecastPage = props => {
...
+                        id: type.intl.id,
+                        defaultMessage: type.intl.defaultMessage,
+                      })}
+                    </SortSelector.Item>
+                  ))}
+                </SortSelector>
+                {currentForecastList.map((obj, index) => (
+                  <QuickForecastCard
+                    forecast={obj}
+                    key={Math.random()}
```
Comment (milklim) [inline src/investarena/components/QuickForecastPage/QuickForecastPage.js]: use 'obj.author + obj.permlink' as a key

File: src/investarena/components/QuickForecastPage/QuickForecastPage.js [hunk 8]
```diff
@@ -40,7 +42,7 @@ const QuickForecastPage = props => {
     setTimeout(() => setLoading(true), 3000);
   }, [props.auth]);
 
-  const getSortItemKey = type => (type.name === 'All' ? '' : type.name);
+  const getSortItemKey = type => (type.name === 'All' ? 'All' : type.name);
```
Comment (milklim) [inline src/investarena/components/QuickForecastPage/QuickForecastPage.js]: we can use just type.name as a sortItemKey

File: src/investarena/components/QuickForecastPage/QuickForecastPage.js [hunk 9]
```diff
@@ -63,146 +65,188 @@ const QuickForecastPage = props => {
...
+                  id: 'how_it_work',
+                  defaultMessage: 'How it works?',
+                })}
+              >
+                <Link to="#" className="rules__link">
+                  {props.intl.formatMessage({
+                    id: 'how_it_work',
+                    defaultMessage: 'How it works?',
+                  })}
+                  &#160;
```
Comment (milklim) [inline src/investarena/components/QuickForecastPage/QuickForecastPage.js]: "&nbsp" is more readable

File: src/investarena/components/QuickForecastPage/QuickForecastPage.js [hunk 10]
```diff
@@ -63,146 +65,188 @@ const QuickForecastPage = props => {
...
-                  endTimerTime={finishRoundTime}
-                  willCallAfterTimerEnd={() => handleFinishTimer()}
-                />
+    <div className="container">
+      <h1 className="head-title">
+        {props.intl.formatMessage({
+          id: 'forecast_title',
+          defaultMessage: 'Guess and get money',
+        })}
+        &#160;
```
Comment (milklim) [inline src/investarena/components/QuickForecastPage/QuickForecastPage.js]: "&nbsp" is more readable

File: src/investarena/components/QuickForecastPage/BallotTimer.js [hunk 11]
```diff
@@ -20,6 +20,8 @@ const BallotTimer = ({ endTimerTime, willCallAfterTimerEnd }) => {
     }
   };
 
+  useEffect(() => () => clearInterval(interval), []);
+
```
Comment (milklim) [inline src/investarena/components/QuickForecastPage/BallotTimer.js]: combine this and next effects. Just return function from next effect.

File: src/client/components/Navigation/Topnav.less [hunk 12]
```diff
@@ -803,4 +804,10 @@ div[data-dir='rtl'] .Topnav {
   border-color: @blue-vivid;
 }
 
+.center-menu {
+  @media screen and (max-width: @medium-bp) and (min-width: @small-bp) {
+    width: 49% !important;
+  }
+}
```
Comment (milklim) [inline src/client/components/Navigation/Topnav.less]: maybe we can use just '@media @small {..'

File: src/client/components/Navigation/Topnav.less [hunk 13]
```diff
@@ -803,4 +804,10 @@ div[data-dir='rtl'] .Topnav {
   border-color: @blue-vivid;
 }
 
+.center-menu {
+  @media screen and (max-width: @medium-bp) and (min-width: @small-bp) {
```
Comment (milklim) [inline src/client/components/Navigation/Topnav.less]: maybe we can use just '@media @Small {..'

File: src/investarena/redux/reducers/forecastReducer.js [hunk 14]
```diff
@@ -84,27 +71,44 @@ export default (state = initialState, action) => {
...
       };
 
     case activeForecastTypes.GET_QUICK_FORECAST_REWARDS.ERROR:
       return state;
 
     case activeForecastTypes.ANSWER_QUICK_FORECAST: {
-      const answeredForecast = state.quickForecastData[action.payload.id];
-      state.quickForecastData.splice(action.payload.id, 1);
+      const answeredForecast = state.quickForecastData.find(
+        frcast => frcast.permlink === action.payload.permlink,
```
Comment (milklim) [inline src/investarena/redux/reducers/forecastReducer.js]: find by author and permlink

**Pull request merged**